### PR TITLE
Fixing missing/broken links in setup instructions

### PR DESCRIPTION
**Pull request by DamienIrving**
Ready for review.

### CODE REVIEW

**Comment by DamienIrving** [comment]
Come to think of it, rather than linking to https://github.com/merely-useful/py-rse/raw/book/data/zipf.zip for people to download the data files, should we put them up on Figshare (or similar site)? That way the dataset has a permanent location, rather than a location that can disappear if we make a tiny change to the name of our repo or even just the `book` branch. We can follow our own advice in the provenance chapter to document the data on Figshare.

**Comment by lwjohnst86** [comment]
Great question @DamienIrving... I'm definitely in favour of submitting to figshare/zenodo. As you said, it would make things much easier for those using the book.

**Comment by gvwilson** [comment]
:+1: to Figshare/Zenodo - practice what we preach.

File: links.md [hunk 1]
```diff
@@ -21,6 +21,7 @@
 [configparser]: https://docs.python.org/3/library/configparser.html
 [covenant]: https://www.contributor-covenant.org
 [creative-commons]: https://creativecommons.org/
+[data-files]: https://github.com/merely-useful/py-rse/raw/book/data/zipf.zip
```
Comment (gvwilson) [inline links.md]: To be changed to Figshare (?)

**Comment by DamienIrving** [comment]
Ok. I've posted the dataset to Figshare and updated the Setting Up appendix accordingly.
https://doi.org/10.6084/m9.figshare.13040516

**Comment by gvwilson** [comment]
You're a good person - is this one ready to merge?

**Comment by DamienIrving** [comment]
Yep, ready to merge.

**Pull request merged**